test/lint.ts

```diff
@@ -199,6 +199,23 @@ const main = async (
 
   if (!hasErrors) {
     console.log(chalk`{green All data {bold passed} linting!}`);
+    if (linters.linters.some((linter) => linter.exceptions)) {
+      console.log(
+        chalk`{yellow Linters have some exceptions, please help us remove them!}`,
+      );
+      for (const linter of linters.linters)
+        if (linter.exceptions) {
+          console.log(
+            chalk`{yellow  ${linter.name} has ${
+              linter.exceptions.length === 1
+                ? '1 excepton:'
+                : `${linter.exceptions.length} exceptions:`
```

Comment:
We can use the `pluralize()` function here:

```suggestion
            chalk`{yellow  ${linter.name} has ${pluralize('exception', linter.exceptions.length)}`
```

Reply:
Thanks, did not know about this!

Reply:
Ah, I missed a bracket, will fix it shortly...